pkg/resources/gadget/gadget.yaml

```diff
@@ -0,0 +1,262 @@
+apiVersion: v1
```

Comment:
I'm not sure it's the best place for this file (same for already existing files). I think we should move the whole `pkg/resources` to a `manifests` folder or similar. (I think we don't expected external users to import this golang package). 

@alban what do you think?

Reply:
I think it is fine to keep it in `pkg/resources`, to keep it simple with `pkg/resources/embed.go`.

It could be called `pkg/resources/manifests/gadget.yaml`

Reply:
Either way works for me. Whatever is the easiest to implement :)